connexion/__init__.py

```diff
@@ -8,7 +8,6 @@
 from .exceptions import ProblemException  # NOQA
 # add operation for backwards compatability
 from .operations import compat
-from .problem import problem  # NOQA
```

Comment:
Raising ProblemExceptions is nicer and much better for everything in core connexion, but removing the ability to use `problem()` by importing directly from `connexion` is a major API change.

Reply:
I agree, we should not break the API without need

Reply:
This had been added back